io.github.milkshiift.GoofCord.yml

```diff
@@ -0,0 +1,70 @@
+app-id: io.github.milkshiift.GoofCord
+
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+
+command: startgoofcord
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: minerals
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 minerals/startgoofcord /app/bin/startgoofcord
+      - install -D minerals/io.github.milkshiift.GoofCord.desktop -t /app/share/applications/
+      - install -D io.github.milkshiift.GoofCord.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: dir
```

Comment:
don't use type dirs. Also this should be part of the same module as the application.